AndroidGesture/app/src/main/java/com/example/ben/androidgesture/GestureUtil.java

```diff
@@ -0,0 +1,22 @@
+package com.example.ben.androidgesture;
+import java.util.*;
+
+/**
+ * Created by Sheldon on 1/31/2017.
+ */
+
+public final class GestureUtil {
+
+
+    public static Gesture recognizeGesture(final List data){
+        // somehow recognize gesture
+        // return whatever direction was detected
+        Gesture gesture = new Gesture(Direction.UP);
+        return gesture;
+    }
+
+    public static void load(){
+        // loads new test data?
+    }
+
+}
```

Comment:
Can you make a utils folder and move this class and all associated classed into it?